detekt/detekt.bzl

```diff
@@ -13,36 +13,36 @@ def _impl(ctx):
         "-Xms16m",
         "-Xmx128m",
         "-cp",
-        ":".join([jar for jar in [ctx.file._detekt_cli_jar.path, ctx.file._detekt_wrapper_jar.path]]),
+        ":".join([jar.path for jar in [ctx.file._detekt_cli_jar, ctx.file._detekt_wrapper_jar]]),
         "io.buildfoundation.bazel.rulesdetekt.wrapper.Main",
     ]
 
     if ctx.attr.config != None:
-        action_inputs.append(ctx.file.config)
+        action_inputs += [ctx.file.config]
```

Comment:
I think this is worse, you allocate a list only to get rid of it

Reply:
Same below

Reply:
Looks better though, at least comparing with `action_arguments` actions. I can change everything to `append` instead of doing `+=`.

Reply:
Yeah I know that it looks better… But for example in our project we have 1800+ detekt targets, imagine amount of allocations that Bazel will have to do during analysis phase if we use `+=` everywhere instead of `append`

I'm fine reading `append` to save some milliseconds for every engineer running this code in big projects :)

Reply:
Fair enough. Made everything `append` and switched the remaining `+` with `extend`.

Reply:
Discussing in Bazel Slack: https://bazelbuild.slack.com/archives/CDCEKVCHY/p1575584159003900